### ISSUE TITLE
Higher level trace service masks lower level trace service.

### ISSUE THREAD

**Issue by husky-parul** [open-telemetry/opentelemetry-go#2081]
### Description

Objective: Enabling tracing in [openshift/kubelet](https://github.com/openshift/kubernetes/pull/778) and [cri-o](https://github.com/cri-o/cri-o/pull/4883/) individually and exporting it to `otlp logging` and `Jaeger`.

### Environment

- OpenShift/Local Kubernetes on Centos -8
- Go Version: 1.16
- opentelemetry-go version: v1.0.0

### Steps To Reproduce

Run instrumented `kubelet` and `cri-o` as `systemd` on the host. 

### Expected behavior

2 trace services capturing kubelet and cri-o calls should be exported to `jaeger` and `oltp logging`

### Current behavior:
ATM only kubelet service is been exported
But when we deploy just cri-o (instrumented), the cri-o traces are exported.

**Comment by Aneurysm9** [comment]
Your deployment topology is unclear.  These appear to be independent applications.  Are they both configured to export to the same destination?  Does one invoke the other via gRPC?

**Comment by husky-parul** [comment]
That's correct. Both `cri-o` and `kubelet` are independent applications and they are running as `systemd` service on the host. They are both configured to export to the same destination, which is  `CollectorGRPCPort = 4317`

`kubelet` communicates with `cri-o` over Unix sockets using the gRPC framework. Please refer to the architecture diagram of cri-o [here](https://cri-o.io/assets/images/architecture.png)

**Comment by sallyom** [comment]
* launch with kubelet instrumented: see kubelet spans
* launch with crio instrumented: see crio spans
* launch with both instrumented: only see kubelet spans, no errors or indication that crio is even instrumented

we've tried: 
* export OTLP both - only see kubelet spans (otel-collector -> jaeger)
* export OTLP crio, stdout export kubelet - only see stdout spans from kubelet (journalctl output)
* export stdout crio, OTLP kubelet - only see OTLP spans from kubelet (otel-collector -> jaeger) 
* export stdout both - only see kubelet stdout (journalctl output)

**Comment by MadVikingGod** [comment]
Just taking a stab at this, it looks like when you launch both that the cri-o isn't exporting at all. 
Would there be any configuration differences that would prevent it from doing that? What I would start checking is that the environment variables are set, if you configure via them; that we aren't seeing any strange SELinux preventing this communication; and that it doesn't launch in a container-like environment.

Right now we are working on a debugging solution, but don't have a good process yet.  But if you aren't getting output from the stdout exporter then there is a very good chance that the sdk isn't running at all.
